src/praisonai-agents/praisonaiagents/runtime/builtin.py

```
@@ -0,0 +1,197 @@
+                metadata = {
+                    'model': getattr(agent, 'model', None),
+                    'agent_id': getattr(agent, 'agent_id', None),
+                    'runtime': self.runtime_id
+                }
+                
+                return RuntimeResult(
+                    content=str(result) if result else "",
+                    metadata=metadata
+                )
+            
+        except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)